src/routers/groupRoutes.js

```diff
@@ -0,0 +1,73 @@
+let express = require("express");
+let router = express.Router();
+const Joi = require("joi");
+const validator = require("express-joi-validation").createValidator({});
+
+let groupModel = require("../models").Groups;
+let GroupService = require("../services/groupService");
+let groupService = new GroupService(groupModel);
+
+const bodySchema = Joi.object().keys({
```

Comment:
Consider moving validation into a separate file next to the related route.
E.g. groupRoutes.validation.js

Consider doing it for ALL the routes.

Reply:
resolved